site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -168,6 +168,7 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.allocator.generateTLS`                      | Set to true to generate TLS certificates or false to provide certificates in `certs/allocator/*`| `true`                 |
 | `agones.allocator.disableMTLS`                      | Turns off client cert authentication for incoming connections to the allocator.            | `false`                |
 | `agones.allocator.disableTLS`                       | Turns off TLS security for incoming connections to the allocator. | `false`                |
+| `agones.allocator.nodeSelector`                     | Allocator [node labels][nodeSelector] for pod assignment                                        | `{}`                   |
```

Comment:
Looks good - but can we drop this in the section below with the header "New Configuration Features".

This means it won't show up in current documentation, and will only show up in development, and the next release :+1: 

#1947 should also unblock CI as well. Thanks!

Reply:
Updated in latest commit.